mount.go

```diff
@@ -126,20 +127,34 @@ func checkMountPoint(dir string) error {
 	return nil
 }
 
-func fusermount(binary string, argv []string, additionalEnv []string, wait bool) (*os.File, error) {
+func fusermount(
+	binary string,
+	argv []string,
+	additionalEnv []string,
+	wait bool,
+	debugLogger *log.Logger) (*os.File, error) {
```

Comment:
Please leave all the function parameters on one line. Splitting the function declaration is not common in Go.

Reply:
Done

Reply:
Yes, I know there are many pre-existing issues in this codebase.

gofmt doesn’t flag/fix all style issues, there’s also https://staticcheck.io/ for example.